managers/oclc_catalog.py

```diff
@@ -13,6 +13,10 @@
 class OCLCCatalogManager:
     CATALOG_URL = 'http://www.worldcat.org/webservices/catalog/content/{}?wskey={}'
     OCLC_SEARCH_URL = 'https://americas.discovery.api.oclc.org/worldcat/search/v2/'
+    ITEM_TYPES = ['archv', 'audiobook', 'book', 'encyc', 'jrnl']
+    LIMIT = 50
+    MAX_NUMBER_OF_RECORDS = 100
```

Comment:
Is this an arbitrary/placeholder number or does this seem like it achieves parity with the old process? Either way, good to have these as constants

Reply:
It definitely will retrieve less than the current implementation. This just speeds its up for testing purposes. I think we can tweak this number accordingly!